### ISSUE TITLE
Add parser for "Listing Quality and Suppressed Listing Report"

### ISSUE THREAD

**Issue by racke** [interchange/Amazon-MWS#26]
Report identifier `_GET_MERCHANT_LISTINGS_DEFECT_DATA_`.

Contents of the report:

Tab-delimited flat file listing quality and suppressed listing report that contains your listing information that is incomplete or incorrect.

Basis call to request record:

```
my $uploader = Amazon::MWS::Uploader->new(%conf);
$uploader->client->RequestReport(ReportType => '_GET_MERCHANT_LISTINGS_DEFECT_DATA_');

```

**Comment by rabbiveesh** [comment]
Don't you think it's beyond the role of this module to parse the report? I can give you the necessary options for Text::CSV, but this module is just for the network access part.

In addition, your example there only requests the report, you still need to call `GetReportRequestList` to poll until your report is ready, and when you get the report id from there, you need to call `GetReport` with that report id to actually download it.

**Comment by racke** [comment]
I don't think it is beyond the role of the module. No point for everyone to invent his own wheels on that.
This task would be the first step. I will add a generic module for report polling later.
